chapter07/pxd/ex1.pl

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use feature "say";
+
+say "Enter a couple of lines (Ctrl-D to exit):";
+
+while (<>) {
+    
+    # Will match 'freddie'
+    if (/fred/) { 
+    	say "The entered line contains the letters fred "; 
+    }
+    
+    # will match only 'fred' 
+    if ( /(^|\s)fred(\s|$)/ ) {
```

Comment:
@thatguy59  - Yeah, the intention was to print the line if it matched the letters and bare word both.
@ardavey  - After going the roundabout way of figuring this out, I realized they were talking about it in the next chapter :confused: